backends/qualcomm/tests/models.py

```diff
@@ -29,7 +29,7 @@ def __init__(self):
         super().__init__()
 
     def forward(self, x):
-        return 10.0 + x
+        return 10 + x
```

Comment:
What's this for? Is it a different test case?

Reply:
It's a minor fix when checking if integer type addition works. Here changes the constant to be integer type to match the function title `AddConstantLong`.